pkg/interface/src/views/apps/publish/app.tsx

```diff
@@ -136,7 +136,10 @@ export default function PublishApp(props: PublishAppProps) {
               const notebookContacts =
                 bookGroupPath in contacts ? contacts[bookGroupPath] : {};
 
-              const notebook = notebooks[ship][book];
+              const notebook = notebooks?.[ship]?.[book];
+              if(!notebook) {
+                return null
+              }
```

Comment:
I'd actually pull this one out and revert it to where it was because the issue is due to `ship` here being the note's author, not the notebook's owner, so it never loads. Blank screens gracefully now if you load a note by a different author.

Reply:
Oh wait. Saw the new commit. Sorry for micromanaging.